sunpy/visualization/animator/base.py

```diff
@@ -435,51 +435,29 @@ def __init__(self, data, image_axes=[-2, -1], axis_ranges=None, **kwargs):
             slider_axes.remove(x)
 
         if len(slider_axes) != self.num_sliders:
-            raise ValueError("Specific the same number of axes as sliders!")
+            raise ValueError("Number of sliders doesn't match the number of slider axes.")
         self.slider_axes = slider_axes
 
         # Verify that combined slider_axes and image_axes make all axes
         ax = self.slider_axes + self.image_axes
         ax.sort()
         if ax != list(range(self.naxis)):
```

Comment:
So we are just checking the number of axes match? So `ax` is just a list of numbers?

Reply:
That's correct.  They list which axes are image axes and which are slider axes.  Does this need a further comment to make this clearer?

Reply:
No this is fine.